website/content/blog/2025-06-17-openbao-to-openssf.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "OpenBao Joins the OpenSSF to Advance Secure Secrets Management in Open Source"
+description: ""
+slug: openbao-joins-the-openssf-to-advance-secure-secrets-management-in-open-source
+authors: OpenSSF
+tags: [announcement, community, collaboration]
+image: https://openssf.org/wp-content/uploads/2025/06/OpenSSF-Blog-Graphics-53-1.png
+---
+
+![OpenBao Joins the OpenSSF to Advance Secure Secrets Management in Open Source](https://openssf.org/wp-content/uploads/2025/06/OpenSSF-Blog-Graphics-53-1.png)
+
+We’re excited to welcome [OpenBao](https://openssf.org/projects/openbao/) to the [Open Source Security Foundation (OpenSSF)](https://openssf.org) as a newly accepted [sandbox project](https://openssf.org/projects/openbao/)!
+
+OpenBao is an open source identity-based secrets and encryption management system that helps organizations securely store, manage, and audit access to sensitive data like API keys, passwords, and certificates. Originally developed under LF Edge and forked from HashiCorp Vault, OpenBao has now found its new home in the OpenSSF, where it aligns more directly with the needs of security professionals and open source maintainers.
+
+“Joining OpenSSF has been a dream come true,” said **Alex Scheel**, Chair of the OpenBao Technical Steering Committee. “This wouldn’t be possible without the support of OpenSSF’s Technical Advisory Council—many thanks for their help and consideration!”
+
+<!-- truncate -->
+
+## Why OpenBao?
+
+Modern systems rely on a growing number of secrets to operate securely—from database credentials to cloud service API tokens. Managing and auditing access to these secrets is complex, and building custom solutions can introduce more risk than reward. OpenBao solves this problem by providing robust encryption services that are tightly gated by authentication and authorization controls. It integrates with the tools and ecosystems open source developers and security teams already use, including Sigstore and SLSA.
+
+Through its CLI, UI, and HTTP API, OpenBao makes it easier to:
```

Comment:
Do we have visual to support this?
A picture of the UI or an architecture diagram of how openbao is typically  used, to better display its role in a cloud setup?